examples/svg-components/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "svg-components",
+  "version": "1.0.0",
+  "scripts": {
+    "dev": "next",
+    "build": "next build",
+    "start": "next start"
+  },
+  "dependencies": {
+    "next": "beta"
+  },
+  "devDependencies": {
+    "babel-plugin-inline-react-svg": "^0.2.0"
```

Comment:
@nkzawa if one needs a SVG in multiple places, one can simply create a file dedicated for it and the import that in multiple places.

E.g. a `icons.js` export multiple components which itself has imported through the babel transform